Makefile

```diff
@@ -79,6 +79,20 @@ build-and-push-multi-arch-debug: build-and-push-container-linux-debug build-and-
 
 push-container: build-container
 
+validate-container-linux-amd64: init-buildx
```

Comment:
Add a comment how these build rules should be used (eg, the validation is done in build-and-push, so these are useful when you want to validate but don't want to figure out / overwrite a $(stagingimage):$(stagingversion)?)

Reply:
Or actually I guess they're used by the verify script.

Reply:
Added a comment about how this is used.